funding/manager.go

```diff
@@ -1295,16 +1343,69 @@ func (f *Manager) handleFundingOpen(peer lnpeer.Peer,
 		return
 	}
 
-	// Only echo back a channel type in AcceptChannel if we actually used
-	// explicit negotiation above.
-	var chanTypeFeatureBits *lnwire.ChannelType
+	var scidFeatureVal bool
```

Comment:
Commenting here since there isn't a diff above, but w.r.t the chan acceptor: we should auto reject if we have the zero conf bit defined and there's no acceptor instance. Otherwise the acceptor should always define if it's permitted or not.